examples/oft-solana/README.md

```diff
@@ -198,31 +198,37 @@ sh -c "$(curl -sSfL https://release.solana.com/v1.17.31/install)"
 
 ### Create the Solana OFT
 
-For OFT:
+:information_source: In all the OFT patterns listed below, the SPL token's Mint Authority is set to the **Mint Authority Multisig**, which always has the **OFT Store** as a signer. The multisig is fixed to needing 1 of N signatures.
+
+:information_source: For **OFT** and **OFT Mint-And-Burn Adapter**, you have the option to specify additional signers through the `--additional-minters` flag. If you choose not to, you must pass in `--only-oft-store true`, which means only the **OFT Store** will be a signer for the \_Mint Authority Multisig\*.
+
+:warning: If you choose to go with `--only-oft-store`, you will not be able to add in other signers/minters or update the Mint Authority.
+
+#### For OFT:
 
 ```bash
 pnpm hardhat lz:oft:solana:create --eid 40168 --program-id <PROGRAM_ID>
 ```
 
-:warning: You may specify the `--additional-minters` flag to add a CSV of additional minter keys to the mint
-multisig. If you do not want to, you must specify `--only-oft-store true`. If you choose the latter approach, you can never
-substitute in a different mint authority.
+:warning: Use `--additional-minters` flag to add a CSV of additional minter addresses to the Mint Authority Multisig. If you do not want to, you must specify `--only-oft-store true`.
 
-For OFTAdapter:
+:information_source: You can also specify `--amount <AMOUNT>` to have the OFT minted to your deployer address upon token creation.
```

Comment:
Can you test this and check if your balance actually increased on deployer account? For me I think I tried it and I wasn't sure if it worked. I'm not sure if it credited my account and created token account.

Reply:
Ran through the 4 various scenarios. When amount is specified, you can see that Mint/MintTo was called and the OFT is minted to my deployer address

no amount specified, onlyOftStore mode https://solscan.io/tx/3ogRs55ZhjEB7hQbDMXqofqjQQ9xgcXW4EzdpzUYWjgZxTYDhwBTTH7q5wiaG9GQVcK1foYeS4fnAGxZUbqEQhuq?cluster=devnet

amount specified, onlyOftStore mode
https://solscan.io/tx/Lap7SZ1n68djiHaHvZR5RS9akUsqpbWMiRFjqmy7zH5p6M7jiFVVpHsnKqobTWfmoDZTss81gaCSrpPnfj1qopn?cluster=devnet

no amount specified, additionalMinters specified
https://solscan.io/tx/ToVzrSW6GP9RpJwvgUXCkAxWLncMRr32XNLKZmkNEDZRheprrrY39Ra6qqdkxQZh3bpG7yBfzjmKDjbZbNX9X5m?cluster=devnet

amount specified, additionalMinters specified
https://solscan.io/tx/29HAx8YAxSEiuJspsfyEQBec1umcQwJhJ8hPD8bvFDzQd8ZowReBX6L2HBdrMbPdubaCNGsnTQacdP8i4c8NpfFu?cluster=devnet